micromamba/src/microserver.cpp

```diff
@@ -1,414 +0,0 @@
-// Copyright (c) Alex Movsisyan
```

Comment:
This file and `server.cpp` are not mentioned in CMake at all, so I think it's safe to remove it.
Please, make sure I'm right.